src/app.coffee

```diff
@@ -36,15 +39,21 @@ defaults =
   driver: "memory"
   params: {}
   address: null,
-  
-if fs.existsSync(CONFIG_FILE)
-  try
-    config = JSON.parse(fs.readFileSync(CONFIG_FILE))
-  catch err
-    console.error err
-    process.exit 1
-else
-  config = {}
+  port: null
+
+if process.env.HOME
+  CONFIG_FILES.push path.join process.env.HOME, ".ofirehose.json"
+
+getConfig = (file) ->
+  if fs.existsSync(file)
+    try
+      config = JSON.parse(fs.readFileSync(file))
+    catch err
+      console.error new Error "Error parse JSON file"
```

Comment:
Creating a new `Error` here isn't super useful - just pass the string directly to `console.error`. And `s/parse/parsing/` :)

Reply:
Also I think we should get rid of the next line and just log the error message on the same line: so something like:

```coffeescript
console.error 'Error parsing JSON file ' + file + ':', err.message

Reply:
better